apps/server/src/modules/management/management-server.module.ts

```diff
@@ -0,0 +1,43 @@
+import { Dictionary, IPrimaryKey } from '@mikro-orm/core';
+import { MikroOrmModule, MikroOrmModuleSyncOptions } from '@mikro-orm/nestjs';
+import { DynamicModule, Module, NotFoundException } from '@nestjs/common';
+import { ALL_ENTITIES } from '@shared/domain';
+import { MongoMemoryDatabaseModule } from '@shared/infra/database';
+import { MongoDatabaseModuleOptions } from '@shared/infra/database/mongo-memory-database/types';
+import { DB_PASSWORD, DB_URL, DB_USERNAME } from '@src/config';
+import { ManagementModule } from './management.module';
+
+export const defaultMikroOrmOptions: MikroOrmModuleSyncOptions = {
+	findOneOrFailHandler: (entityName: string, where: Dictionary | IPrimaryKey) => {
```

Comment:
How many copies of this code we already have? :P

Reply:
we need ticket ;o)